src/dev/code_coverage/ingest_coverage/team_assignment/index.js

```diff
@@ -20,29 +20,39 @@
 import { run } from '@kbn/dev-utils';
 import { TEAM_ASSIGNMENT_PIPELINE_NAME } from '../constants';
 import { fetch } from './get_data';
-import { noop } from '../utils';
 import { update } from './update_ingest_pipeline';
 
-export const uploadTeamAssignmentJson = () => run(execute, { description });
-
 const updatePipeline = update(TEAM_ASSIGNMENT_PIPELINE_NAME);
 
-function execute({ flags, log }) {
+const execute = ({ flags, log }) => {
   if (flags.verbose) log.verbose(`### Verbose logging enabled`);
 
-  fetch().fold(noop, updatePipeline(log));
+  const logLeft = handleErr(log);
+  const updateAndLog = updatePipeline(log);
+
+  const { path } = flags;
+
+  fetch(path).fold(logLeft, updateAndLog);
```

Comment:
What will happen now if `getContents` fails?

Reply:
logLeft() should happen.